build.sbt

```diff
@@ -1,4 +1,4 @@
-val cogcompNLPVersion = "3.0.30"
+val cogcompNLPVersion = "3.0.37"
```

Comment:
and you need to make some changes in the way evaluators are called. Now we don't have nay `setViews` functions. Instead we pass the views whenever we call the `evaluate` function. See this PR: 
https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/pull/58/files